plugins/base/src/test/kotlin/enums/EnumsTest.kt

```diff
@@ -20,29 +25,112 @@ class EnumsTest : BaseAbstractTest() {
             }
         }
 
+        val writerPlugin = TestOutputWriterPlugin()
+
         testInline(
             """
             |/src/main/kotlin/basic/Test.kt
-            |package enums
+            |package testpackage
```

Comment:
Maybe it would be convenient to have a help function like
```kt
    @Test
    fun testEnum(code: String, block: T.() -> Unit) {
        val configuration = dokkaConfiguration {
            sourceSets {
                sourceSet {
                    sourceRoots = listOf("src/")
                }
            }
        }

        val writerPlugin = TestOutputWriterPlugin()

        testInline(
            """
            |/src/main/kotlin/basic/Test.kt
            |package testpackage
            |$code
            """.trimMargin(),
            configuration,
            pluginOverrides = listOf(writerPlugin),
            block
        ) 
    }
```

Reply:
Indeed. Enum tests generally need some refactoring and more test cases, I don't want to get too much into it in this PR.

I'm sure we'll get to improving enum support in the future, so let's refactor it then